setup.py

```diff
@@ -30,5 +30,8 @@ def get_version():
         # 'Development Status :: 7 - Inactive',
         'Operating System :: POSIX',
         'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
@i-trofimtschuk Do we need `3.6` ?

Reply:
@kaloyanm why not? this is what we test on travis